_posts/2020-07-14-mill-brook-summer-2020.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Mill Brook Summer 2020
+description: As we all try and adjust to the changes each day brings, the beauty of nature always awaits us outside. Here are some photos of our community gardens, wildlife, and the Path’s calming vistas. For everyone’s safety and protection, please wear a mask while out walking. Thank you.
+---
+<img src = "assets/images/millbrooksummer20201.fnageercomexport.jpg.png" />
```

Comment:
please change the following with the image:

- [ ] include one image in the post header as that will show up in the list pages correctly
- [ ] rename the images to exclude `fnageercomexport`. that can imply copyright of the images which you do not have
- [ ] why is there both `jpg` and `png` extension? switch to the correct mimetype of the images
- [ ] images need to be included via markdown. so `![alt text](link)`. see any markdown cheetsheet for more examples. note that the `alt text` is important as it allows things to be indexed better by search enginges
- [ ] path of the images is wrong. you need to link to it via jinja template filter. something like ``{{ assets/.../.../png | relative_url }}`. as it is now the images wont render on the post page:
  
  <img width="1157" alt="image" src="https://user-images.githubusercontent.com/932940/87500859-9e4d9780-c62b-11ea-942f-196309c8a94a.png">